symengine/lib/symengine_wrapper.pyx

```diff
@@ -1458,6 +1556,14 @@ cdef class Complex(Number):
 
 class Infinity(Number):
 
+    @property
+    def is_infinite(self):
+        return True
+
+    @property
+    def is_prime(self):
```

Comment:
I'm just wondering why having this here would be useful at all if `is_prime` is defined for classes with only `False`.